packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -52,7 +52,9 @@
       applicationName: string().required("Your application must have a name."),
     },
     {
-      username: string().required("Your application needs a first user."),
```

Comment:
Given now that the user's `name` property can't be implicitly set from the `username` (if no name for the user was entered) we should probably require a name for the user in our validation schemas.